google-api-client/pom.xml

```diff
@@ -110,7 +111,6 @@
     <dependency>
       <groupId>com.google.http-client</groupId>
       <artifactId>google-http-client-gson</artifactId>
-      <scope>test</scope>
```

Comment:
This still adds a new dependency on gson. I think you may want to use the JsonFactory interface from google-http-java-client.